languages/tree-sitter-stack-graphs-java/src/stack-graphs.tsg

```diff
@@ -185,28 +359,95 @@ attribute node_symbol = node            => symbol = (source-text node), source_n
   edge member -> def
 }
 
+(method_declaration (formal_parameters (_) @param)) @method
+{
+  edge @param.lexical_scope -> @method.lexical_scope
+  edge @method.lexical_scope -> @param.def
+  edge @method.defs -> @param.def
+}
+
+(formal_parameter type: (_) @type (_) @id) @param
+{
+  node @param.def
+  node @param.lexical_scope
+  node def__typeof
+
+  edge @type.lexical_scope -> @param.lexical_scope
+
+  attr (@param.def) node_definition = @id
```

Comment:
Per @hendrikvanantwerpen's review of #152, maybe this variable should be `@name` too.